performance_regression_test.py

```diff
@@ -38,6 +38,52 @@ def display_single_result(self, result):
                                           result['latency 95th percentile'],
                                           result['latency 99th percentile'],
                                           result['latency 99.9th percentile']))
+        f = open('jenkins_perf_PerfPublisher.xml', 'w')
+        content = """<report name="simple_regression_test report" categ="none">
+
+  <test name="simple_regression_test" executed="yes">
+    <description>"simple regression test"</description>
+    <targets>
+      <target threaded="yes">target-%s</target>
+      <ami_id>%s</ami_id>
+      <stress_modes>%s</stress_modes>
+    </targets>
+    <platform name="scylla_platform">
+      <instance_type_db>%s</instance_type_db>
+    </platform>
+
+    <result>
+      <success passed="yes" state="1"/>
+      <performance unit="kbs" mesure="%s" isRelevant="true" />
```

Comment:
Are you sure that this property is really spelled `mesure` instead of `measure`?